src/lwc/rdEntryForm/rdEntryForm.html

```diff
@@ -0,0 +1,159 @@
+<template>
+   
+    <lightning-record-edit-form record-id={recordId}
+                                object-api-name="npe03__Recurring_Donation__c"
+                                onload={handleLoad}
+                                onsubmit={handleSubmit}
+                                onerror={handleError}
+                                onsuccess={handleSuccess}>
+        <template if:true={isLoading}>
+            <div class="slds-grid slds-grid_align-center">
+                <div class="slds-is-relative slds-size_1-of-1">
+                    <lightning-spinner alternative-text="Loading"></lightning-spinner>
+                    <c-util-stencil row-count="20" column-count="2"></c-util-stencil>
+                </div>
+            </div>
+        </template>
+
+        <div data-id="editFields" class="slds-hide">
+
+            <template if:true={hasError}>
```

Comment:
I will be adding comments individually for some feedback since I cannot see all previous comments in the PR review mode. 
Thanks for adding padding on the modal! The error message requires padding still. Also, the Currency field is less indented than other fields in the same column.